website/docs/reference/meta-cli/configuration-file.mdx

```diff
@@ -17,6 +20,16 @@ typegraphs:
       migrations_path: "migrations"
 ```
 
+## Typegate configuration
+Each typegate node is configured under `typegates.<name>`.
+The typegate node selection is done by the `--target` option in the CLI.
+
```

Comment:
I believe there might be other location to be updated in the doc

Reply:
Yes, but we don't yet have the reference for the CLI...

Reply:
@Natoandro https://metatype.dev/docs/guides/securing-requests#authentication maybe on other place (like tutorial)